model/build.gradle

```diff
@@ -75,9 +75,71 @@ task generateSpdxLicenseList {
     }
 }
 
+task generateSpdxLicenseExceptionList {
+    doLast {
+        println "Fetching SPDX license list..."
+
+        def jsonSlurper = new groovy.json.JsonSlurper()
+        def spdxUrl = "https://raw.githubusercontent.com/spdx/license-list-data/master/json/exceptions.json".toURL()
+        def json = jsonSlurper.parse(spdxUrl)
+        def spdxLicenseExceptionIds = json.exceptions.collect {
+            it["licenseExceptionId"]
+        }
+
+        println "Found ${spdxLicenseExceptionIds.size()} SPDX license exception identifiers."
+
+        def licenseExceptionEnumFile = file('src/main/kotlin/config/SpdxLicenseException.kt')
+        licenseExceptionEnumFile.write("""\
+            /*
```

Comment:
Note to myself: In a later commit we could move this to a function that also replaces "2018" with the current year.

Reply:
Yes, this also needs more cleanup as there is also too much duplication currently, but as there are more SPDX related changes coming I first wanted to see watch other changes are required before optimizing the code.